scripts/rff/preprocess_csv_test.py

```diff
@@ -0,0 +1,20 @@
+"""Tests for preprocess_csv.py"""
+
+import os, sys, unittest
+
+# Allows the following module imports to work when running as a script
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from rff import preprocess_csv
+
+class ProcessTest(unittest.TestCase):
+    def test_preprocess(self):
+        src_fldr = f"test_data/input/prism/daily/county"
+        output_csv = f"test_data/output/WeatherVariability_Counties.csv"
```

Comment:
the output_csv can perhaps go into a temp directory?

e.g., from https://github.com/datacommonsorg/data/blob/master/scripts/glims/rgi/process_test.py

```
  with tempfile.TemporaryDirectory() as tmp_dir:
    output_csv = f'{tmp_dir}/WeatherVariability_Counties.csv'
    ...
```